code/modules/antagonists/hivemind/spells/hivemind_panic.dm

```diff
@@ -0,0 +1,56 @@
+/obj/effect/proc_holder/spell/targeted/induce_panic
+	name = "Induce Panic"
+	desc = "We unleash a burst of psionic energy, inducing a debilitating fear in those around us and reducing their combat readiness. We can also briefly affect silicon-based life with this burst."
+	panel = "Hivemind Abilities"
+	charge_max = 900
+	range = 7
+	invocation_type = "none"
+	clothes_req = 0
+	max_targets = 0
+	antimagic_allowed = TRUE
+	action_icon = 'icons/mob/actions/actions_hive.dmi'
+	action_background_icon_state = "bg_hive"
+	action_icon_state = "panic"
+
+/obj/effect/proc_holder/spell/targeted/induce_panic/cast(list/targets, mob/living/user = usr)
+	var/datum/antagonist/hivemind/hive = user.mind.has_antag_datum(/datum/antagonist/hivemind)
+	if(!hive)
+		to_chat(user, "<span class='notice'>This is a bug. Error:HIVE1</span>")
+		return
+	for(var/mob/living/carbon/human/target in targets)
+		if(target.stat == DEAD)
+			continue
+		target.Jitter(14)
+		target.apply_damage(35 + rand(0,15), STAMINA, target.get_bodypart(BODY_ZONE_HEAD))
+		if(is_hivehost(target))
+			continue
+		if(prob(20))
+			var/text = pick(";HELP!","I'm losing control of the situation!!","Get me outta here!")
+			target.say(text, forced = "panic")
+		if(prob(1))
+			SEND_SOUND(target, sound('sound/effects/adminhelp.ogg'))
```

Comment:
lol

Reply:
90% of these things you are pointing out are all the old ability code I didn't look at properly, god damnit